feat(tem): scale plan MTA-5622 (#4576)
scaleway/docs-content: faq/transactional-email.mdx

Yes, you can manually unblock any address, even if it is still under the temporary block period. You can unblock up to five addresses every 24 hours.

## What happens if an address on a TEM blocklist becomes valid again?

> You can manually unblock any addresses. Make sure you confirm that the initial cause of the bounce was resolved before doing so to avoid further issues.

Reviewer: ```suggestion
You can manually unblock the address. Before doing so, ensure that the initial cause of the bounce has been resolved to prevent further issues.
```

## Can I remove addresses from the blocklist?

> Yes, all blocked addresses can be unblocked via the interface. For addresses you added manually, you have full control over when and if they are removed from the blocklist.

Reviewer: ```suggestion
Yes, all blocked addresses can be unblocked via the interface. If you manually added an address, you have full control over when and if it is removed from the blocklist.
```

Reviewer: Intrerface = scaleway console ?

## What are the main benefits of the Scale plan?

> The Scale plan provides advanced control over email deliverability through a dedicated IP, detailed performance tracking, and advanced webhook management. It is ideal for businesses with high email sending volume requirements.

Reviewer 2: ```suggestion
The Scale plan provides advanced control over email deliverability through a dedicated IP, detailed performance tracking, and advanced webhook management. It is ideal for businesses with high email-sending volume requirements.
```

## Can I use both a dedicated IP and a shared IP on the same project?
No, a project using a dedicated IP cannot send emails via a shared IP. If you want to use a shared IP, you must create a separate project.

Refer to the [Understanding managed dedicated IPs](/transactional-email/reference-content/tem-dedicated-ip) documentation page for more information.

## Does a dedicated IP include both IPv4 and IPv6?
Yes, each dedicated IP provided with the Scale plan includes both an IPv4 and an IPv6 address. This ensures better compatibility with different email providers and optimizes the deliverability of sent emails.

Refer to the [Understanding managed dedicated IPs](/transactional-email/reference-content/tem-dedicated-ip) documentation page for more information.

## Can I add a new dedicated IP to my Organization?

> If you want to add a new dedicated IP to your Organization, you must create a new project with the Scale plan or contact support to review your request.

Reviewer 3: ```suggestion
If you want to add a new dedicated IP to your Organization, you must create a new Scaleway Project with the Scale plan or contact support to review your request.
```

Refer to the [Understanding managed dedicated IPs](/transactional-email/reference-content/tem-dedicated-ip) documentation page for more information.

## Does Scaleway have an anti-spam policy?

> Yes. You can find Scaleway official Transaction Email anti-spam policy in the [contracts](https://www.scaleway.com/en/contracts/) section of the Scaleway website.

Reviewer 3: ```suggestion
Yes. You can find Scaleway's official Transactional Email anti-spam policy in the [contracts](https://www.scaleway.com/en/contracts/) section of the Scaleway website.
```

> Yes. You can find Scaleway official Transaction Email anti-spam policy in the [contracts](https://www.scaleway.com/en/contracts/) section of the Scaleway website.

Reviewer 2: ```suggestion
Yes. You can find Scaleway's official Transaction Email anti-spam policy in the [contracts](https://www.scaleway.com/en/contracts/) section of the Scaleway website.
```
